docgen/parser/parser.go

```diff
@@ -220,16 +220,39 @@ func parseMethods(pkgName string, messages map[string]*Message, methods []*googl
 			return nil, err
 		}
 
+		example, err := generateMethodExample(messages, m.GetOutputType())
+		if err != nil {
+			return nil, err
+		}
+
 		res[getQualifiedName(pkgName, m.GetName())] = &Method{
 			Name:      m.GetName(),
+			Request:   req,
 			Response:  messages[m.GetOutputType()],
+			Example:   example,
 			Operation: extOp.(*options.Operation),
-			Request:   req,
 		}
 	}
 	return res, nil
 }
 
+func generateMethodExample(messages map[string]*Message, name string) (string, error) {
```

Comment:
nit: `generateResponseExample` with `Method` field named `ResponseExample` would be clearer (at this point we might want to make `Method`'s `Response` field a type of its own)

Reply:
did that in latest commit.